### DISCUSSION TITLE
Preact - is the transition still happening?

### DISCUSSION THREAD

**Discussion by neanda** [forem/forem#18433] [Feature Requests]
Hi there, you mention that you're transitioning to Preact, is this still an active process and do you have an intended date?  I didn't find anything here in the discussions.

Sorry that I cannot contribute to the code (I'm a UX designer).  This is a 'very' interesting product and I'd like to see it progress, it's definitely something needed.  Thank you.

**Comment by Ridhwana** [comment]
Hi @neanda 👋 

Thank you for reaching out. 

Our architecture includes Preact currently, whereby we follow the [Islands Architecture](https://jasonformat.com/islands-architecture/) pattern: we render as much as we can server-side, and then we use Preact to enhance the parts of that server-rendered page that require richer interactivity. 

You can read more about it [here](https://developers.forem.com/frontend/preact) and find some of the Preact components [here](https://github.com/forem/forem/tree/main/app/javascript).
